modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -561,14 +571,18 @@ conf = {{{packageName}}}.Configuration(
             {{/servers}}
         ]
 
-    def get_host_from_settings(self, index, variables=None):
+    def get_host_from_settings(self, index, variables=None, servers=None):
         """Gets host URL based on the index and variables
         :param index: array index of the host settings
         :param variables: hash of variable and the corresponding value
+        :param servers: host settings for the endpoint
```

Comment:
Can you describe that servers should is of type (list/None) here?

Reply:
This method should be private in first place. I might create a private version of this method instead.